x-pack/solutions/observability/plugins/streams/server/lib/streams/root_stream_definition.ts

```diff
@@ -34,3 +36,8 @@ export const rootStreamDefinition: WiredStreamDefinition = {
     },
   },
 };
+
+export function hasSupportedStreamsRoot(streamName: string) {
+  const root = streamName.split('.')[0];
```

Comment:
nit: There is the `getSegments` helper for this

Reply:
Fixed in https://github.com/elastic/kibana/pull/208914/commits/b8ac05de59ffdc8e56ed5e0f2fc89bceb46900c9